docs/modules/ROOT/pages/api/fungibleToken.adoc

```diff
@@ -11,17 +11,35 @@ TIP: For an overview of the module, read the {fungible-guide}.
 
 [.hljs-theme-dark]
 ```ts
-import "./node-modules/@openzeppelin-compact/fungible-token/FungibleToken" prefix FungibleToken_;
+import "./node-modules/@openzeppelin-compact/fungible-token/FungibleToken";
 ```
 
 [.contract]
-[[FungibleToken]]
+[[FungibleTokenModule]]
 === `++FungibleToken++` link:https://github.com/OpenZeppelin/compact-contracts/blob/main/contracts/fungibleToken/src/FungibleToken.compact[{github-icon},role=heading-link]
```

Comment:
I'm curious why the section name is `FungibleToken` but all the links are `FungibleTokenModule`.

Reply:
That's a fair point. I was thinking that `FungibleTokenModule` future proofs the anchors if/when we add extensions or other core modules because we have `FungibleToken` as the main doc header as well. We could change the section name to `FungibleTokenModule`, but since the actual module name is just `FungibleToken`, I vote we keep that consistent

Do you have a preference or other ideas for organizing here?